src/v2/components/Favorites/Accounts.jsx

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React from 'react';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import cn from 'classnames';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import {useTheme} from '@material-ui/core/styles';
+import {observer} from 'mobx-react-lite';
+import {Link} from 'react-router-dom';
+import {map} from 'lodash/fp';
+import HelpLink from 'v2/components/HelpLink';
+
+import useStyles from './styles';
+
+type THead = {
```

Comment:
please rename to 'TableHead' or whatever the unabbreviated is? when I first saw this I thought it was a typo + miscapitalization of 'thread'

Reply:
done